libs/domain/picking/src/components/picking/picking.component.ts

```diff
@@ -305,6 +310,25 @@ export class PickingComponent extends PickingListMixin(LitElement) {
     `;
   }
 
+  protected override updated(): void {
+    this.onTabChange();
+  }
+
+  protected onTabChange(): void {
+    for (let i = 0; i < this.tabRefs.length; i++) {
+      const tab = this.tabRefs[i].value;
+      if (!tab) {
+        continue;
+      }
+      const chip = tab.querySelector('oryx-chip');
```

Comment:
Should we use ref again to not query directly by name('oryx-chip'), in case somehow in the future this could change ?